modules/apps/layout/layout-admin-web/src/main/java/com/liferay/layout/admin/web/internal/util/comparator/FriendlyURLEntryLocalizationComparator.java

```diff
@@ -12,7 +12,7 @@
  * details.
  */
 
-package com.liferay.friendly.url.web.internal.util.comparator;
+package com.liferay.layout.admin.web.internal.util.comparator;
```

Comment:
you are making this comparator to layout only, but extending . why?

but letting other kind of objects, removing : 
```
!Objects.equals(_getClassName(httpServletRequest),
					Layout.class.getName())
```
					
get the friendly URL... I do not understand